pyproject.toml

```diff
@@ -138,12 +138,12 @@ manylinux-x86_64-image = "manylinux2014"
 manylinux-i686-image = "manylinux2014"
 skip = "pp* cp36-* cp37-* *musllinux* *win32 *i686 cp38-macosx_arm64"
 test-skip = "*win32 *linux_i686"
-test-command = "python {project}/examples/python/stochastic_swap.py"
+test-command = "stestr run --abbreviate"
```

Comment:
Does the path for this work correctly? `stestr` only works if it's in the repository root with it's config file.